libevmasm/AssemblyItem.h

```diff
@@ -147,6 +149,14 @@ class AssemblyItem
 		solAssert(_tag.type() == Tag);
 		return AssemblyItem(ConditionalRelativeJump, Instruction::RJUMPI, _tag.data(), _debugData);
 	}
+	static AssemblyItem swapn(size_t _depth, langutil::DebugData::ConstPtr _debugData = langutil::DebugData::create())
```

Comment:
I guess you could treat it as a single word, and there's no other helper like this here to set a precedent, but I'd go with `swapN`/`dupN`, because in many places we treat these opcode names as consisting of multiple terms and use camel case on them.

```suggestion
	static AssemblyItem swapN(size_t _depth, langutil::DebugData::ConstPtr _debugData = langutil::DebugData::create())
```

Reply:
cameel case  you mean

Reply:
For the record, I'm partial to snake case :P